core/sql/executor/ExExeUtilLoad.cpp

```diff
@@ -1778,6 +1811,19 @@ short ExExeUtilHBaseBulkLoadTcb::moveRowToUpQueue(const char * row, Lng32 len,
 }
 
 
+short ExExeUtilHBaseBulkLoadTcb::printLoggingLocation(int bufPos)
+{
+  short retBufPos = bufPos;
+  if (hblTdb().getNoOutput())
+    return 0;
+  if (loggingLocation_ != NULL) {
+     str_sprintf(&statusMsgBuf_[bufPos], "       Logging Location: %s", loggingLocation_);
```

Comment:
How do we know the target buffer is long enough? Is there some limit on the length of loggingLocation_?